### PR TITLE
phy: phy-cadence-dp: Rename Cadence DP PHY driver

### PR DESCRIPTION
**Pull request by yamonkarcdns**
Rename the Cadence PHY driver from phy-cadence-dp to phy-cadence-torrent
for future use although currently the driver only has DisplayPort support.
Modified Makefile, Kconfig and audio_display.cfg accordingly.

Signed-off-by: Yuti Amonkar <yamonkar@cadence.com>

### CODE REVIEW

File: ti_config_fragments/audio_display.cfg [hunk 1]
```diff
@@ -34,7 +34,7 @@ CONFIG_DRM_CDNS_MHDP=m
 CONFIG_DRM_CDNS_DSI=n
 
 CONFIG_PHY_J721E_WIZ=y
-CONFIG_PHY_CADENCE_DP=y
+CONFIG_PHY_CADENCE_TORRENT=y
```
Comment (tomba) [inline ti_config_fragments/audio_display.cfg:38]: Create a separate commit for changing the ti_config_fragments. They will never go upstream, so it's better to keep them in separate commits.

**Review by tomba — CHANGES_REQUESTED** [review]
Looks fine, but this just changes the file name. The code itself still uses DP PHY terms, e.g. the compatible string is "cdns,dp-phy". Wouldn't it be better to do all these changes together in one series? Or is there a reason you want to rename the file first?

**Comment by yamonkarcdns** [comment]
Created separate branch cdns-phy-torrent-v2 for the requested changes.
